pandas/core/indexes/base.py

```diff
@@ -1089,6 +1101,12 @@ def astype(self, dtype, copy: bool = True):
         --------
         numpy.ndarray.take: Return an array formed from the
             elements of a at the given indices.
+
+        Examples
+        --------
+        >>> idx = pd.Index(['a', 'b', 'c'])
+        >>> idx.take([2])
```

Comment:
I think `take` is often used with more than one element, and there can be repetitions - maybe something like
```
In [10]: idx = pd.Index(['a', 'b', 'c'])

In [11]: idx.take([2, 2, 1, 2])
Out[11]: Index(['c', 'c', 'b', 'c'], dtype='object')
```

Reply:
Done

Reply:
It's green @MarcoGorelli